app/controllers/reminders/edit.js

```diff
@@ -0,0 +1,11 @@
+import Ember from 'ember';
+
+export default Ember.Controller.extend({
+  actions: {
+    updateReminder() {
+      this.transitionToRoute('reminders')
```

Comment:
I don't understand this. The data is updating. How can the model not be saved if the reminder changes persist thru page changes?